### PR TITLE
Add more detailed version to tests results slack message

### PR DESCRIPTION
**Pull request by karola312**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [x] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

## [Optional] Description

This PR will add more details about version on slack tests results. It is minor - I don't need these changes only on main branch

### CODE REVIEW

**Review by Cloud11PL — APPROVED** [review]

File: .changeset/rich-dryers-repair.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+More datailed version in tests rusults message on slack is displayed
```
Comment (poulch) [inline .changeset/rich-dryers-repair.md:5]: typo, and I'd rephrase it
``` suggestion
Tests results slack message now show detailed version of .....
```

**Review by NyanKiyoshi — APPROVED** [review]

**Review by poulch — APPROVED** [review]
